systest/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM    docker-registry.pdbld.f5net.com/kevin/jenkins-worker-ubuntu-16.04:9d1fcc866de21a1a6c2c761a5829fc9ddb2da51a
```

Comment:
This is temporary.  This file is not used by test infrastructure yet.   I _do_ want it in the commit, because it's the right pattern to document the specification here, (going forward).   This change should not affect the way tests run currently.